build.psm1

```diff
@@ -211,6 +211,10 @@ function Start-PSBuild {
         [switch]$NoPSModuleRestore,
```

Comment:
@rjmholt, your last commit had 4 failures in `PowerShell-CI-macos`
Error occurred in test script '/Users/vsts/agent/2.155.1/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1'
```powershell
An AST cannot be used as the child of more than one AST. To use this AST in another AST, call the Copy() method and use its result.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.155.1/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1'
```powershell
An AST cannot be used as the child of more than one AST. To use this AST in another AST, call the Copy() method and use its result.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.155.1/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1'
```powershell
An AST cannot be used as the child of more than one AST. To use this AST in another AST, call the Copy() method and use its result.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.155.1/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1'
```powershell
An AST cannot be used as the child of more than one AST. To use this AST in another AST, call the Copy() method and use its result.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```